src/components/Designs.vue

```diff
@@ -0,0 +1,339 @@
+                        @click="showAllGeneKnockouts = true"
+                        :hidden="showAllGeneKnockouts"
+                      >
+                        ...
+                      </a>
+                    </div>
+                  </div>
+                </td>
+              </tr>
+            </template>
+          </v-data-table>
+        </div>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { mapGetters } from "vuex";
+
+export default Vue.extend({
+  name: "Designs",
+  data: () => ({
+    selected: [],
+    expand: true,
+    showAllReactionKnockins: false,
+    showAllReactionKnockouts: false,
+    showAllGeneKnockouts: false,
+    isDeletionDialogVisible: false,
+    isLoading: false,
+    headers: [
+      { text: "Name", value: "name", width: "20%" },
+      { text: "Organism", value: "organism_id", width: "15%" },
+      { text: "Model", value: "model_id", width: "15%" },
+      { text: "Added reactions", value: "reaction_knockins", width: "15%" },
+      { text: "Reaction knockouts", value: "reaction_knockouts", width: "15%" },
+      { text: "Gene knockouts", value: "gene_knockouts", width: "15%" }
+    ],
+    pagination: {
+      rowsPerPage: 10
+    }
+  }),
+  methods: {
+    customSort(items, index, isDesc) {
+      items.sort((a, b) => {
+        if (
+          index === "reaction_knockins" ||
+          index === "reaction_knockouts" ||
+          index === "gene_knockouts"
+        ) {
+          if (!isDesc) {
+            return a["design"][index].length - b["design"][index].length;
+          }
+          return b["design"][index].length - a["design"][index].length;
+        }
+        if (index === "organism_id") {
+          if (!isDesc) {
+            return this.organism(this.model(a.model_id).organism_id).name <
+              this.organism(this.model(b.model_id).organism_id).name
+              ? -1
+              : 1;
+          }
+          return this.organism(this.model(b.model_id).organism_id).name <
+            this.organism(this.model(a.model_id).organism_id).name
+            ? -1
+            : 1;
+        }
+        if (!isDesc) {
+          return a[index] < b[index] ? -1 : 1;
+        }
+        return b[index] < a[index] ? -1 : 1;
+      });
+      return items;
+    },
+    getReactionId(reaction, method) {
+      if (method === "Pathway") {
+        return JSON.parse(reaction).bigg_id;
+      }
+      return reaction;
+    },
+    reactionLink(reaction, method, isReactionKnockin) {
+      const reactionId =
+        isReactionKnockin && method === "Pathway"
+          ? JSON.parse(reaction).bigg_id
+          : reaction;
+      if (reactionId.startsWith("MNX")) {
+        return `https://www.metanetx.org/equa_info/${reactionId}`;
+      }
+      return `http://bigg.ucsd.edu/search?query=${reactionId}`;
+    },
+    geneLink(geneId) {
+      return `http://bigg.ucsd.edu/search?query=${geneId}`;
```

Comment:
Not a big deal but I don't really feel this needs to be a method, just put the URL directly in the `href`.